website/siteConfig.js

```diff
@@ -7,6 +7,28 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+/* List of talks & videos */
+
+const videoTypes = {
+  YOUTUBE: 'youtube',
+  IFRAME: 'iframe',
+};
+
+const videos = [
+  {
+    title: 'Learn how to test JavaScript with Jest',
```

Comment:
Can we swap this one out with a different one?

Reply:
Oh, I thought it was removed from main page only to be included here.